### PR TITLE
Added organization and space filtering for applist

### PR DESCRIPTION
**Pull request by shsamkit**
Allow to filter apps by organization and/or space

### CODE REVIEW

**Comment by giner** [comment]
Hey @shsamkit, thank for the PR!

File: cf-applist.sh [hunk 1]
```diff
@@ -41,7 +43,7 @@ Usage: $(basename "$0") [OPTION]...
   -n                        ignore cache
   -N                        do not format output and keep it tab-separated (useful for further processing)
   -j                        print json (filter and sort options are not applied when -j is in use)
-  -v                        verbose
+  -v                        verbose
```
Comment (giner) [inline cf-applist.sh:46]: [nit] extra space

File: cf-applist.sh [hunk 2]
```diff
@@ -275,8 +298,13 @@ json_stacks=$(get_json "$next_url" | jq "{stacks:.}")
 next_url="/v3/isolation_segments?per_page=100"
 json_isolation_segments=$(get_json "$next_url" | jq "{isolation_segments:.}")
 
+### Custom part
+# Set organization and space
+org_guid="8a646d58-f913-4344-8006-8a436413beb4"
+space_guid="0728c36a-5907-40d3-966a-88f5787530c3"
```
Comment (giner) [inline cf-applist.sh:304]: Clean up? ^

File: cf-applist.sh [hunk 3]
```diff
@@ -175,6 +184,20 @@ if [[ -n $opt_updated_minutes_older_than ]]; then
                  select (\$updated_min_ago != null) | select (\$updated_min_ago > $opt_updated_minutes_older_than) |"
 fi
 
+# Organization filter
+# If organization is not found, command exits with error-code 1
+ORG_FILTER=""
+if [[ ! -z $opt_org ]]; then
+    ORG_FILTER="&q=organization_guid:`cf org $opt_org --guid`"
```
Comment (giner) [inline cf-applist.sh:191]: ORG and SPACE have to be URL-encoded, see https://github.com/rakutentech/cf-tools/blob/102139b/cf-target-app.sh#L32

[nit, style] use `$()` insead of backticks

**Comment by giner** [comment]
@shsamkit These options seem to be useful. Would you update other *list.sh scripts also?

**Comment by shsamkit** [comment]
@giner , I will check check and add this option wherever possible.

**Comment by shsamkit** [comment]
@giner , I checked the API docs and looks like only [app list](https://apidocs.cloudfoundry.org/9.5.0/apps/list_all_apps.html) supports the space and organization parameters. 

cf-routelist.sh -> [list-all-routes](https://apidocs.cloudfoundry.org/9.5.0/routes/list_all_routes.html) 
cf-routemappings.sh -> [route-mappings](https://apidocs.cloudfoundry.org/9.5.0/routes_mapping/list_all_route_mappings.html) 
They don't support filtering

**Pull request closed without merging**